emission/analysis/userinput/matcher.py

```diff
@@ -63,32 +63,88 @@ def handle_single_most_recent_match(confirmed_trip, ui):
 
 def handle_multi_non_deleted_match(confirmed_trip, ui):
     logging.debug(f"handling user input {ui} for {confirmed_trip}")
```

Comment:
Nothing in this is trip specific, so maybe we should rename the variable to `confirmed_object` or `confirmed_trip_or_place` for consistency

Reply:
Agreed.
I've been using "timeline entry" as a generic term for either a trip or place object

Reply:
Resolved in https://github.com/shankari/e-mission-server/pull/4/commits/5cdbdeb219c85714684677ea9f91f083397141be

I went with `confirmed_object`, but I am also good to change it to `timeline_entry` if we feel that is a better name